src/App.jsx

```diff
@@ -50,13 +53,44 @@ function App() {
       <div>
         <DoiBadge doi_id="ab-cd" />
       </div>
+      <DoiBadge
+        doi="custom-doi/10.ab.cd"
+        label="custom-label"
+        color="#a2e5b7"
+      />
+      <span>ToggleSwitches</span>
       <div>
-        <DoiBadge
-          doi="custom-doi/10.ab.cd"
-          label="custom-label"
-          color="#a2e5b7"
+        <ToggleSwitchBig switchLength="40px" />
+      </div>
+      <div>
+        <ToggleSwitchBig
+          switchLength="120px"
+          onLabel="LT2"
+          offLabel="LT1"
+          fontSize="25px"
+        />
+      </div>
+      <div>
+        <ToggleSwitchBig
+          switchLength="120px"
+          onLabel="Conventional"
+          offLabel="Primitive"
+          fontSize="10px"
         />
       </div>
+      <div style={{ padding: "8px" }}>
```

Comment:
Here, padding will mess up a bit the red border, that is supposed to show the exact borders of the component. Use margin here.